config-model/src/main/java/com/yahoo/searchdefinition/derived/AttributeFields.java

```diff
@@ -153,6 +154,12 @@ public Iterator attributeIterator() {
         return Collections.unmodifiableCollection(attributes.values());
     }
 
+    public Collection<Attribute> structFieldAttributes(String baseFieldName) {
+        return attributes().stream()
+                .filter(attribute -> attribute.getName().startsWith(baseFieldName + "."))
```

Comment:
Even though your mother will probably optimize this one for you I consider it a good habit to do
String structPrefix = baseFieldName + ".";
and then use that in the filter

Reply:
Fixed